test/files/presentation/callcc-interpreter.check

```diff
@@ -3,7 +3,11 @@ reload: CallccInterpreter.scala
 askTypeCompletion at CallccInterpreter.scala(51,34)
 ================================================================================
 [response] askTypeCompletion at (51,34)
-retrieved 66 members
+retrieved 70 members
+[inaccessible] private[this] val self: callccInterpreter.type
+[inaccessible] private[this] val self: callccInterpreter.type
+[inaccessible] private[this] val self: callccInterpreter.type
+[inaccessible] private[this] val self: callccInterpreter.type
```

Comment:
Do you know why these 4x `self` show up now?

Reply:
Ah, thanks. I don't have the background why `[inaccessible]` symbols are included in completions. I wonder if inaccessible symbols of implicit conversion targets should really be there as well, or if that is by mistake.

The `scala$Predef$any2stringadd$$self` name is an "expanded" name set by `sym.makeNotPrivate`, `sym.hasFlag(EXPANDEDNAME)` should be true here.

Reply:
The `[inaccessible]` means that it doesn't actually show up in the completions. These completions generally _do_ include results that need an implicit conversion, so I think this is the right behavior.